File: Excel-Sheet-Column-Number.py
class Solution(object):
    def titleToNumber(self, columnTitle):
        \\\
        :type columnTitle: str
        :rtype: int
        \\\
        x = 0
        for i in columnTitle:
            x = x * 26 + (ord(i) - ord('A') + 1)
        return x

File: Excel-Sheet-Column-Number.js
/**
 * @param {string} columnTitle
 * @return {number}
 */
var titleToNumber = function (columnTitle) {
    let x = 0;
    let n = columnTitle.length;
    for (let i = 0; i < n; i++) {
        x = x * 26 + (columnTitle.charCodeAt(i) - 'A'.charCodeAt(0) + 1);
    }
    return x;
};
